### DISCUSSION TITLE
hi how do i get months in renderer function?

### DISCUSSION THREAD

**Discussion by eitai** [ndresx/react-countdown#199] [Q&A] [answered]
how do i get months in renderer function? 
![image](https://user-images.githubusercontent.com/35004992/174475572-64117dac-d71c-46f6-935a-7e42db8e7fcc.png)
![image](https://user-images.githubusercontent.com/35004992/174475576-2820e64a-571c-4d76-9524-a35faa9ad36e.png)

**Comment by ndresx** [comment, marked as answer]
Hi, I would suggest setting [`daysInHours`](https://github.com/ndresx/react-countdown#daysinhours) to `true` and calculating the remaining months, basically how many hours are making up for a month.

**Reply by eitai** [reply]
1 problem is that i want to display all of them so i will need to calculate everything, i want to make it clean as possible

**Reply by ndresx** [reply]
Sure, you could leave `daysInHours` out of the equation and use the `total` attribute, which is also an argument of the object the `renderer` callback receives to get to the months. Based on that, you can get to months, days, years, and whatever else you might need.

You can also look at https://github.com/ndresx/react-countdown/blob/master/src/utils.ts#L72 for inspiration to see how the available values are calculated and derive your solution for the remaining values from it.

**Reply by eitai** [reply]
Hey, thanks for your time I already managed to do that right  :) thanks again
